### PR TITLE
fix deploy by release's fake runtime data

### PR DESCRIPTION
**Pull request by QvodSoldier**
#### What type of this PR
/kind bug

#### What this PR does / why we need it:
fake runtime should only appear when pipeline is initializing

#### Which issue(s) this PR fixes:
- [Erda Cloud Issue Link](https://terminus-org.app.terminus.io/erda/workBench/projects/387/issues/all?id=74180&issueFilter__urlQuery=eyJzdGF0ZUJlbG9uZ3MiOlsiT1BFTiIsIldPUktJTkciLCJXT05URklYIiwiUkVPUEVOIiwiUkVTT0xWRUQiXSwiYXNzaWduZWVJRHMiOlsiMTAwMDQzNiJdfQ%3D%3D&issueViewGroup__urlQuery=eyJ2YWx1ZSI6ImthbmJhbiIsImNoaWxkcmVuVmFsdWUiOnsia2FuYmFuIjoiZGVhZGxpbmUifX0%3D&iterationID=429&type=BUG)

#### Specified Reviewers:

/assign @Effet

### CODE REVIEW

**Comment by Effet** [comment]
provide a screenshot you fixed it.

File: modules/orchestrator/utils/pipeline_utils.go [hunk 1]
```diff
@@ -134,24 +137,78 @@ func FindCreatingRuntimesByRelease(appID uint64, env string, ymlName string, bdl
...
+					LastOperator:    fmt.Sprintf("%v", v.Extra.RunUser.ID),
+				})
+			}
 		}
 	}
 
 	return result, nil
 }
+
+func isUndoneDeployByReleaseTask(piplineDetail *apistructs.PipelineDetailDTO) bool {
```
Comment (Effet) [inline modules/orchestrator/utils/pipeline_utils.go:207]: what does `UndoneDeploy` mean?

**Review by Effet — CHANGES_REQUESTED** [review]

**Comment by QvodSoldier** [comment]
deploy by release will give user a fake runtime until the real runtime is created
Kapture 2021-07-20 at 15 54 33

File: modules/orchestrator/services/runtime/runtime.go [hunk 2]
```diff
@@ -1257,10 +1257,12 @@ func (r *Runtime) List(userID user.ID, orgID uint64, appID uint64, workspace, na
 			continue
 		}
 		env := strutil.ToLower(runtime.Workspace)
-		if _, exists := rtEnvPermMark[env]; !exists {
-			rtEnvPermMark[env].HaveRuntime = false
+		if _, exists := rtEnvPermBranchMark[env]; !exists {
```
[the remark this answers is not in the document]
Comment (QvodSoldier) [reply modules/orchestrator/services/runtime/runtime.go:1262]: done

File: modules/orchestrator/utils/pipeline_utils.go (hunk 1 above)
Comment (QvodSoldier) [reply modules/orchestrator/utils/pipeline_utils.go]: The function name has been modified and commented

**Review by Effet — APPROVED** [review]

**Pull request merged**